hbase-common/src/main/java/org/apache/hadoop/hbase/util/ImmutableByteArray.java

```diff
@@ -51,4 +51,8 @@ public static ImmutableByteArray wrap(byte[] b) {
   public String toStringUtf8() {
     return Bytes.toString(b);
   }
+
+  public String toStringBinary() {
+    return Bytes.toStringBinary(b);
+  }
```

Comment:
Yeah, why not have this just be the toString behavior rather than add a new method?

Reply:
Because toString and toStringBinary have different behavior, we use the above method in our code base so do not want to change it.

Reply:
When would we ever want a String that had binary non-printables in it? What good is a toString that just does default Object#toString?